spt/OrangeBox/scripts/srctas_reader.cpp

```diff
@@ -459,6 +459,9 @@ namespace scripts
 		propertyHandlers["changelevel"] = &SourceTASReader::HandleCLCondition;
 		propertyHandlers["velyaw"] = &SourceTASReader::HandleVelYaw;
 		propertyHandlers["velpitch"] = &SourceTASReader::HandleVelPitch;
+#if SSDK2007
+		propertyHandlers["portal_bubble"] = &SourceTASReader::HandlePBubbleCondition;
```

Comment:
```suggestion
		if (DoesGameLookLikePortal())
		{
			propertyHandlers["portal_bubble"] = &SourceTASReader::HandlePBubbleCondition;
		}
```
I'm not sure if spt-serverplugin.hpp is included but the function is declared there iirc.